p2p/chain.py

```diff
@@ -331,53 +352,43 @@ def _test() -> None:
     import argparse
     import signal
     from p2p import ecies
-    from p2p import kademlia
-    from p2p.constants import ROPSTEN_BOOTNODES
-    from p2p.discovery import DiscoveryProtocol
     from evm.chains.ropsten import RopstenChain, ROPSTEN_GENESIS_HEADER
     from evm.db.backends.level import LevelDB
     from tests.p2p.integration_test_helpers import FakeAsyncChainDB, LocalGethPeerPool
-    logging.basicConfig(level=logging.INFO, format='%(levelname)s: %(message)s')
-    logging.getLogger('p2p.chain.ChainSyncer').setLevel(logging.DEBUG)
 
     parser = argparse.ArgumentParser()
     parser.add_argument('-db', type=str, required=True)
     parser.add_argument('-local-geth', action="store_true")
+    parser.add_argument('-debug', action="store_true")
     args = parser.parse_args()
 
+    logging.basicConfig(
+        level=logging.INFO, format='%(asctime)s %(levelname)s: %(message)s', datefmt='%H:%M:%S')
+    log_level = logging.INFO
+    if args.debug:
+        log_level = logging.DEBUG
+    logging.getLogger('p2p.chain.ChainSyncer').setLevel(log_level)
+
     loop = asyncio.get_event_loop()
     chaindb = FakeAsyncChainDB(LevelDB(args.db))
     chaindb.persist_header(ROPSTEN_GENESIS_HEADER)
     privkey = ecies.generate_privkey()
     if args.local_geth:
         peer_pool = LocalGethPeerPool(ETHPeer, chaindb, RopstenChain.network_id, privkey)
-        discovery = None
     else:
-        listen_host = '0.0.0.0'
-        listen_port = 30303
-        addr = kademlia.Address(listen_host, listen_port, listen_port)
-        discovery = DiscoveryProtocol(privkey, addr, ROPSTEN_BOOTNODES)
-        loop.run_until_complete(discovery.create_endpoint())
-        print("Bootstrapping discovery service...")
-        loop.run_until_complete(discovery.bootstrap())
-        peer_pool = PeerPool(ETHPeer, chaindb, RopstenChain.network_id, privkey, discovery)
+        from p2p.peer import HardCodedNodesPeerPool
+        peer_pool = HardCodedNodesPeerPool(ETHPeer, chaindb, RopstenChain.network_id, privkey)
+        peer_pool.min_peers = 5
```

Comment:
Given this monkeypatch (and the known need for this to be configurable) can this go ahead and get moved to something we can set in the `__init__`?